docs/appendices/release-notes/5.7.3.rst

```diff
@@ -48,4 +48,9 @@ See the :ref:`version_5.7.0` release notes for a full list of changes in the
 Fixes
 =====
 
-None
+- Fixed an issue that prevented sub-columns of a view to be shown in the
+  ``information_schema.columns`` table if only top-level column was used in
+  view's SELECT query. Workaround: explicitly select sub-columns in the view
+  definition, e.g.::
+
```

Comment:
Workaround exists but treated it as a bug because user, accessing a view, might be not aware about full table schema. Also, sub-columns can be added after view definition.